File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/Frequency.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class Frequency {
  @BeanProperty var count: Long = 0L
  @BeanProperty var year: Int = 0
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class Frequency {\n")
    sb.append("  count: ").append(count).append("\n")
    sb.append("  year: ").append(year).append("\n")
    sb.append("}\n")
    sb.toString
  }
}

File: samples/wordnik-api-java/src/main/java/com/wordnik/client/model/Frequency.java
package com.wordnik.client.model;

public class Frequency {
  private long count = 0L;
  private int year = 0;
  public long getCount() {
    return count;
  }
  public void setCount(long count) {
    this.count = count;
  }

  public int getYear() {
    return year;
  }
  public void setYear(int year) {
    this.year = year;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Frequency {\n");
    sb.append("  count: ").append(count).append("\n");
    sb.append("  year: ").append(year).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
